Plots/Scatter/NCL_scatter_4.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
The resulting plot from this script doesn't match the NCL version.

Reply:
Using the center=True kwarg in rolling corrected the discrepancy. 
![image](https://user-images.githubusercontent.com/46687291/73094576-179ad780-3e9e-11ea-9a1d-bcb46f72e82b.png)

Reply:
Awesome! But there is still a discrepancy in the plots. The Python version has points on the left and right side that aren't in the NCL version.

Reply:
Corrected by adjusting the min_periods kwarg

![image](https://user-images.githubusercontent.com/46687291/73095841-af99c080-3ea0-11ea-908a-81c3d5c94d41.png)